src/main/java/TechJobs.java

```diff
@@ -58,13 +58,13 @@ public static void main (String[] args) {
                 String searchField = getUserSelection("Search by:", columnChoices);
 
                 // What is their search term?
-                System.out.println("\nSearch term:");
+                System.out.println("\nSearch term: ");
```

Comment:
Be careful with println, it adds an extra space before this which is causing some tests to fail I believe